management/src/main/java/org/ehcache/management/ManagementRegistryServiceConfiguration.java

```diff
@@ -46,6 +48,8 @@
    */
   String getCollectorExecutorAlias();
 
+  Collection<String> getTags();
```

Comment:
javadoc ? for example : 
  /**
   * Gets the tags set by the user on the Management service
   *
   * @return The static colector executor alias
   */